Library/Formula/gtk+.rb

```diff
@@ -27,6 +27,13 @@ class Gtkx < Formula
     cause "Undefined symbols when linking"
   end
 
+  # Patch to allow Freeciv's gtk2 client to run.
+  # See Homebrew/homebrew-games#278
+  patch do
```

Comment:
Can you include a link to the upstream bug, please? Since it hasn't been merged yet we'll want to keep an eye on it.